ch.md

```diff
@@ -0,0 +1,20 @@
+### Ch CLI shell
```

Comment:
You can actually just rename the file to `Ch-Shell.md` and get rid the section header. Filename will be displayed at the top of the page anyway.

Adding a reference to this page in the [sidebar](https://github.com/Microsoft/ChakraCore-wiki/blob/master/_Sidebar.md) would also be helpful for visibility, but I can do that after we merge your change.

Reply:
I worry `shell` might be misleading because `ch` has no REPL (not interactive). Simply `ch CLI` might be a more apt title.